cmd/trace/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/crossplaneio/crossplane-cli/pkg/trace"
+
+	"k8s.io/client-go/discovery/cached/disk"
+
+	"k8s.io/client-go/restmapper"
+
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+)
+
+func main() {
+	var help bool
+	var kubeconfig string
+	var namespace string
+	var outputFormat string
+	if home := homedir.HomeDir(); home != "" {
+		pflag.StringVar(&kubeconfig, "kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) Absolute path to the kubeconfig file")
+	} else {
+		pflag.StringVar(&kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file")
+	}
+
+	pflag.BoolVarP(&help, "help", "h", false, "Shows this help message")
+	pflag.StringVarP(&namespace, "namespace", "n", "default", "Namespace")
+	pflag.StringVarP(&outputFormat, "outputFormat", "o", "", "Output format. One of: dot")
+	pflag.Parse()
+
+	if help {
+		printHelp()
+		os.Exit(0)
+	}
+
+	resource := pflag.Arg(0)
+	resourceName := pflag.Arg(1)
+	if resource == "" || resourceName == "" {
+		failWithMessage("missing arguments: SUBCOMMAND TYPE[.GROUP] NAME [-n|--namespace NAMESPACE]")
```

Comment:
I'm noticing this message doesn't match the one printed by usage . . . would it possible for us to reuse the same message? What do you think of the pattern of printing the full usage when the arguments are wrong? That's usually the pattern I go with